contracts/common/interfaces/ILidoLocator.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+
+// SPDX-License-Identifier: GPL-3.0
+
+// See contracts/COMPILERS.md
+// solhint-disable-next-line
+pragma solidity 0.4.24||0.8.9;
+
+interface ILidoLocator {
+    function getLido() external view returns(address);
+    function getDepositSecurityModule() external view returns (address);
+    function getELRewardsVault() external view returns (address);
+    function getOracle() external view returns (address);
```

Comment:
there will be two oracles (accounting and validator exit bus, see contracts/0.8.9/oracle)

Reply:
it's better to disambiguate them and make two separate getters